simpleclient_servlet/src/test/java/io/prometheus/client/exporter/ExampleBenchmark.java

```diff
@@ -0,0 +1,52 @@
+package io.prometheus.client.exporter;
+
+import io.prometheus.client.Gauge;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.UUID;
+
+public class ExampleBenchmark {
+
+    public static void main(String[] args) throws Exception {
+
+        Gauge gauge = Gauge.build().name("labels").help("foo").labelNames("bar").register();
+        for (int i = 0; i < 10000; i++) {
+            gauge.labels(UUID.randomUUID().toString()).set(Math.random());
+        }
+
+        ServletContextHandler context = new ServletContextHandler();
+        context.setContextPath("/");
+        context.addServlet(new ServletHolder(new MetricsServlet()), "/metrics");
+
+        Server server = new Server(1234);
```

Comment:
Do we actually need to run a HTTP server to test this?

Reply:
This hasn't been addressed.

Reply:
It's required since bottleneck is container-specific. For example, with Jetty 8.1.7 the bottleneck seems to be the ByteArrayBuffer instances. But with Jetty 9.4.27, ByteArrayBuffer is no longer an issue but another part of the code is consuming CPU cycles. In either case, the wrapper patch improved the performance by approximately the same amount.

Reply:
Can we at least bind to 0 so we don't clash with any existing listeners?